src/app/loans/loans-view/loan-account-actions/loan-account-actions.component.html

```diff
@@ -3,3 +3,9 @@
     <mifosx-loans-account-close *ngIf="actions.close"></mifosx-loans-account-close>
   </mat-card>
 </div>
+
+<div class="container">
+  <mat-card>
+    <mifosx-undo-disbursal *ngIf="actions.undodisbursal"></mifosx-undo-disbursal>
```

Comment:
```suggestion
    <mifosx-undo-disbursal *ngIf="actions.undodisbursal"></mifosx-undo-disbursal>
```
This can be added in the previous card itself.